grails-app/domain/org/pih/warehouse/shipping/Shipment.groovy

```diff
@@ -671,5 +671,9 @@ class Shipment implements Comparable, Serializable {
         Event event = events.find { Event event -> event?.eventType?.eventCode == EventCode.CUSTOMS_RELEASE }
         return event?.eventDate
     }
+
+    Boolean hasInvoicedItem() {
```

Comment:
nitpicky: this could be a primitive `boolean` (`shipmentItems` can't be null)

Reply:
@jmiranda agreed with you. I wouldn't mind (and actually I don't mind, that's why I've approved the PR without this change) the `Boolean` if the value could be null - in this case it can't - no matter what would happen.
I think I've raised this topic once there (or actually it was about naming): https://github.com/openboxes/openboxes/pull/4266#discussion_r1316064617

With that said I'd also be happy to discuss it, because I have read a lot of discussions, about this topic (as it's always been bothering me) with pros and cons of using both versions without a clear answer what's the best.

Reply:
@kchelstowski Yeah I think the larger discussion around using wrappers (Integer) vs primitives (int) is probably one that should be investigated, discussed on a tech huddle, and added to our coding conventions. I've added that to a ticket for coding convention topics that need to be discussed https://pihemr.atlassian.net/browse/OBS-1406.